tests/drivers/i2s/i2s_additional/prj.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_I2S=y
+CONFIG_ZTEST=y
+
+CONFIG_LOG=y
+CONFIG_LOG_MODE_MINIMAL=n
```

Comment:
Why is it needed (a brief comment would be useful here)? `LOG` is anyway enabled by `ZTEST`.